pkg/occlient/fakeclient.go

```diff
@@ -0,0 +1,32 @@
+package occlient
+
+import (
+	fkappsclientset "github.com/openshift/client-go/apps/clientset/versioned/fake"
+	fkbuildclientset "github.com/openshift/client-go/build/clientset/versioned/fake"
+	fkimageclientset "github.com/openshift/client-go/image/clientset/versioned/fake"
+	fkprojectclientset "github.com/openshift/client-go/project/clientset/versioned/fake"
+	fkrouteClientset "github.com/openshift/client-go/route/clientset/versioned/fake"
+	fkkubernetes "k8s.io/client-go/kubernetes/fake"
+)
+
+// FkClientSet : hold fake ClientSets
+type FkClientSet struct {
+	kubeClientset    *fkkubernetes.Clientset
+	imageClientset   *fkimageclientset.Clientset
+	appsClientset    *fkappsclientset.Clientset
```

Comment:
why are there clients if they are never initialized? This will just lead to unnecessary errors.